### DISCUSSION TITLE
Fetch log cannot see the full request urls in terminal (all end with ellipsis)

### DISCUSSION THREAD

**Discussion by idaaaaaaa** [vercel/next.js#51387] [App Router]
![image](https://github.com/vercel/next.js/assets/132628076/a838ba4a-cd68-42b0-a46b-0c34b0300197)
How can I see the full request urls in terminal, currently no matter how big my terminal window is, the urls will always have an ellipsis

**Comment by mengnans** [comment]
Having the same issue

**Comment by h-adamik** [comment]
Same here, very inconvenient. Makes it pretty useless when some part of the URL is missing
